src/Pact/Types/Runtime.hs

```diff
@@ -200,6 +201,8 @@ data ExecutionFlag
   | FlagDisablePact48
   -- | Disable Pact 4.9 Features
   | FlagDisablePact49
+  -- | Enable verifiers
+  | FlagEnableVerifiers
```

Comment:
It feels odd that we have both FlagDisable and FlagEnable variants. My guess is that everything should have been FlagEnable, but do we want the dissonance until then?

Reply:
Usually we have a `FlagDisablePactXY` - I'd prefer that here. This stuff will go into mainnet.